functions/lib/stripe/webhooks.py

```
@@ -348,42 +336,62 @@ def complete_fulfilment_session_request(
+                timeout=30,
+            )
+
+            if not response.ok:
+                error_response = response.json()
+                error_message = error_response.get(
+                    "errorMessage", f"HTTP {response.status_code}"
+                )
+                logger.error(
+                    f"complete_fulfilment_session: Cloud function error: Failed to complete fulfilment session: {error_message}"
+                )
+                raise Exception(f"complete_fulfilment_session: {error_message}")
```

Ruff (0.13.1)

Create your own exception

(TRY002)